src/v1/members/routes.rs

```diff
@@ -37,6 +50,85 @@ pub(super) async fn get_members(
     ))
 }
 
+fn enriched_member(xmpp_service: &XmppService, jid: &JID) -> EnrichedMember {
+    trace!("Enriching `{jid}`…");
+
+    let vcard = match xmpp_service.get_vcard(jid) {
+        Ok(Some(vcard)) => Some(vcard),
+        Ok(None) => {
+            debug!("`{jid}` has no vCard.");
+            None
+        }
+        Err(err) => {
+            // Log error
+            error!("Could not get `{jid}`'s vCard: {err}");
+            // But dismiss it
+            None
+        }
+    };
```

Comment:
We retrieve the user's vCard. If it fails (e.g. unknown user) or the user has no vCard, we do as if they had no vCard and return `null` fields.